Assets/Scenes/Level2.unity

```diff
@@ -554,14 +554,14 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 5bc3a628348bea4408c521ac7ec602ab, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
-  noneMessage: 
-  magniferMessage: 
-  brushMessage: 
-  materialMessage: 
-  coveredMessage: 
-  magnifierClip: {fileID: 0}
-  materialClip: {fileID: 0}
-  tipManager: {fileID: 0}
+  noneMessage: NA
+  magniferMessage: The handel is covered in leather strips wich deyed a black color.
```

Comment:
awesome! just a few typos:
"The handle is covered in leather strips which are dyed black."